bundle/libraries/local_path.go

```diff
@@ -66,6 +66,11 @@ func IsLibraryLocal(dep string) bool {
 }
 
 func isPackage(name string) bool {
+	// If the dependency has ==, it's a package with version
+	if strings.Contains(name, "==") {
+		return true
+	}
+
```

Comment:
Is there a spec for the lines in `requirements.txt` we can follow to get the exhaustive list of options?

I.e. if it is possible to use `./foobar==foo.whl` this will bail again.